### DISCUSSION TITLE
New Back Office field TagType

### DISCUSSION THREAD

**Discussion by luigimassa** [PrestaShop/PrestaShop#33253] [Ideas]
### The problem

In symfony form and in PrestaShop there isn't any  standard field TagType

### The solution

It's possible to implement a TagType field integrating these library

- https://sliptree.github.io/bootstrap-tokenfield/
- https://tagin.netlify.app/?tags=uno%2Cdue%2Ctre

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Hlavtox** [comment]
@luigimassa Hi, did you check the product page for example? Isn't this what you want? :-)
![Snímek obrazovky 2023-07-14 155010](https://github.com/PrestaShop/PrestaShop/assets/6097524/293c5266-20c6-4076-a66c-0cf07220b994)

**Reply by MatShir** [reply]
We should add to the UI kit
